services/retrieve_and_rank/v1.js

```diff
@@ -213,6 +432,30 @@ module.exports = function (RED) {
       password: {type:"password"}
     }
   });
+  RED.nodes.registerType('watson-retrieve-rank-cluster-settings', clusterSettingsNode, {
+    credentials: {
+      username: {type:"text"},
+      password: {type:"password"}
+    }
+  });
+  RED.nodes.registerType('watson-retrieve-rank-upload-solr-configuration', uploadSolrConfigurationNode, {
+    credentials: {
+      username: {type:"text"},
+      password: {type:"password"}
+    }
+  });
+  RED.nodes.registerType('watson-retrieve-rank-solr-configuration-settings', solrConfigurationSettingsNode, {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2405715173/issues/source?bid=3225115&fileBranchId=3233651#l447)